File: 0001_0100/0059/0059.py
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    n, k = (int(x) for x in f.read().split())
    m, s = 1, 0
    while n:
        i = n % k
        s += i
        m *= i
        n //= k
    q.write(str(m - s))

        
    
           
        

File: 0001_0100/0059/0059.go
package main

import (
	"fmt"
	"os"
)

func main() {
	f, _ := os.Open("input.txt")
	q, _ := os.Create("output.txt")
	var n, k, i, s int
	fmt.Fscan(f, &n, &k)
	m := 1
	for n > 0 {
		i = n % k
		s += i
		m *= i
		n /= k
	}
	fmt.Fprint(q, m-s)
}
